### DISCUSSION TITLE
zellij run with multiple commands from command line

### DISCUSSION THREAD

**Discussion by codezninja** [zellij-org/zellij#3459] [Q&A]
I'm trying to integrate gh-dash with zellij so when I open a PR in gh dash to review with key bindings I can open a pane / tab and run a few commands. ie `zellij run -f --cwd {{.RepoPath}} -- "git stash -u && gh pr checkout {{.PrNumber}} && nvim .`.

I'm unable to running multiple commands with zellij run. Is there a better way to do this? I'm still very new to zellij so still learning alot about it in docs and haven't messed with custom layouts yet which I think might be a way but haven't figured out how to send in dynamic data

**Comment by Lockszmith-GH** [comment]
try the good all shell:

```shell
zellij run -f --cwd {{.RepoPath}} -- "sh -c 'git stash -u && gh pr checkout {{.PrNumber}} && nvim .'"
```
